transport-native-epoll/src/main/java/io/netty/channel/unix/DomainSocketChannel.java

```diff
@@ -30,4 +31,6 @@
 
     @Override
     DomainSocketChannelConfig config();
+
+    PeerCredentials peerCredentials() throws IOException;
```

Comment:
@Scottmitch how you feel about adding this method here ? Its a breaking change so maybe not add it at all to the interface for now and only add it to the actual implementation would be the best. WDYT ?